src/main/java/core/basesyntax/db/FruitStorage.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax.db;
+
+import core.basesyntax.model.Fruit;
+import java.util.HashMap;
+import java.util.Map;
+
+public class FruitStorage {
+    public static final Map<Fruit, Integer> storage = new HashMap<>();
```

Comment:
just a recommendation, not to use in Map structure mutable classes, or make your own immutable with all rules needed